packages/office-addin-cli/.eslintrc.json

```diff
@@ -1,8 +1,23 @@
 {
-  "plugins": [
-    "office-addins"
-  ],
-  "extends": [
-    "plugin:office-addins/recommended"
-  ]
-}
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint", "prettier"],
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "project": "./tsconfig.json"
+    },
+    "extends": ["eslint:recommended"],
+    "rules": {
+      "prettier/prettier": ["error", { "endOfLine": "auto" }],
+      "no-eval": "error",
+      "no-delete-var": "warn",
+      "no-octal": "warn",
+      "no-inner-declarations": "warn",
+      "no-unused-vars": "off",
+      "@typescript-eslint/no-unused-vars": "error"
+    }
+  }
+  
```

Comment:
Where did this set come from?

Reply:
I took it from our `eslint-plugin-office-addins` module.